documentation/docs/steps/abapAddonAssemblyKitCheckCVs.md

```diff
@@ -0,0 +1,47 @@
+# ${docGenStepName}
+
+## ${docGenDescription}
+
+## Prerequisites
+
```

Comment:
You could think about including user/authorization requirements here.

Reply:
I added an remark that user/pw can be supplied, too - hope this was what you ment?

Reply:
Ok, in the File View I now got that I should enter something in the Prerequisites section, not further down the file... added a remark that the credentials must be put in the jenkins credential store